websites/C/Chess/presence.ts

```diff
@@ -1,4 +1,4 @@
-var presence = new Presence({
+let presence = new Presence({
```

Comment:
const

Reply:
fix it at other places in the code too. No var :/

Reply:
Oops. Ok I will fix that.